package.json

```diff
@@ -35,6 +36,6 @@
     "url": "git+https://github.com/Originate/exosphere.git"
   },
   "scripts": {
-    "postinstall": "morula all 'yarn install'"
+    "postinstall": "./yarn-install.sh"
```

Comment:
This won't work on Windows. Can you write a LiveScript file that lives in `bin` and does this?